pb/terrarium/provider/provider.proto

```diff
@@ -84,11 +84,11 @@ message ProviderRequest {
   string arch = 4;
 }
 
-message UploadProviderSourceZipRequest {
+message UploadProviderBinaryRequest {
   ProviderRequest provider = 1;
```

Comment:
How will the request identify the provider arch / os combination the provider request already contains a list of repeated platform items. that will not enable you to specify how a specific upload is for a given os / arch. How are you planning to map binary with an architecture?